bundles/org.openhab.binding.miio/src/main/resources/ESH-INF/thing/vacuumThing.xml

```diff
@@ -127,6 +129,16 @@
 		<category>Energy</category>
 		<state pattern="%.0f" readOnly="true" />
 	</channel-type>
+	<channel-type id="water_box_status">
+		<item-type>Switch</item-type>
+		<label>Water Box State</label>
```

Comment:
Isn't this value in the app called Water Level? in that case it may be better to call it similar here

Reply:
No, the water box status is not shown in the app at all. You confuse this with the `"water_box_custom_mode"`.

The `"water_box_custom_mode"` has not yet been implemented: https://github.com/openhab/openhab-addons/issues/6628